recipes/faiss/meta.yaml

```
@@ -0,0 +1,233 @@
+    'gpu/impl/L2Norm.cuh', 'gpu/impl/L2Select.cuh', 'gpu/impl/Metrics.cuh',
+    'gpu/impl/PQCodeDistances-inl.cuh', 'gpu/impl/PQCodeDistances.cuh', 'gpu/impl/PQCodeLoad.cuh',
+    'gpu/impl/PQScanMultiPassNoPrecomputed-inl.cuh', 'gpu/impl/PQScanMultiPassNoPrecomputed.cuh',
+    'gpu/impl/PQScanMultiPassPrecomputed.cuh', 'gpu/impl/VectorResidual.cuh',
+    'gpu/utils/DeviceMemory.h', 'gpu/utils/DeviceUtils.h', 'gpu/utils/MemorySpace.h',
+    'gpu/utils/StackDeviceMemory.h', 'gpu/utils/StaticUtils.h', 'gpu/utils/Timer.h',
+    'gpu/utils/BlockSelectKernel.cuh', 'gpu/utils/Comparators.cuh',
+    'gpu/utils/ConversionOperators.cuh', 'gpu/utils/CopyUtils.cuh', 'gpu/utils/DeviceDefs.cuh',
+    'gpu/utils/DeviceTensor-inl.cuh', 'gpu/utils/DeviceTensor.cuh', 'gpu/utils/DeviceVector.cuh',
+    'gpu/utils/Float16.cuh', 'gpu/utils/HostTensor-inl.cuh', 'gpu/utils/HostTensor.cuh',
+    'gpu/utils/Limits.cuh', 'gpu/utils/LoadStoreOperators.cuh', 'gpu/utils/MathOperators.cuh',
+    'gpu/utils/MatrixMult-inl.cuh', 'gpu/utils/MatrixMult.cuh', 'gpu/utils/MergeNetworkBlock.cuh',
+    'gpu/utils/MergeNetworkUtils.cuh', 'gpu/utils/MergeNetworkWarp.cuh',
+    'gpu/utils/NoTypeTensor.cuh', 'gpu/utils/Pair.cuh', 'gpu/utils/PtxUtils.cuh',
+    'gpu/utils/ReductionOperators.cuh', 'gpu/utils/Reductions.cuh', 'gpu/utils/Select.cuh',
+    'gpu/utils/Tensor-inl.cuh', 'gpu/utils/Tensor.cuh', 'gpu/utils/ThrustAllocator.cuh',
+    'gpu/utils/Transpose.cuh', 'gpu/utils/WarpSelectKernel.cuh', 'gpu/utils/WarpShuffles.cuh'
+] %}
+
+package:
+  name: faiss-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/faiss/archive/v{{ version }}.tar.gz
+  sha256: e1a41c159f0b896975fbb133e0240a233af5c9286c09a28fde6aefff5336e542
+  patches:
+    - patches/0001-change-python-Makefile-to-point-to-libfaiss.-SHLIB_E.patch
+    - patches/0002-remove-fPIC-flag-for-msvc-compat.patch  # [win]
+
+build:
+  number: {{ number }}
+  # TODO: figure out compiler errors on windows
+  skip: true                      # [win]
+  # TODO: remove this to retry building GPU on the feedstock
+  skip: true                      # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+outputs:
+  - name: faiss-proc
+    version: 1.0.0
+    build:
+      string: {{ faiss_proc_type }}
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/faiss-feedstock
+      license: MIT
+      license_family: MIT
+      summary: A meta-package to select CPU or GPU build for faiss.
+
+  - name: libfaiss
+    script: build-lib.sh          # [not win]
+    script: build-lib.bat         # [win]
+    build:
+      string: "h{{ PKG_HASH }}_{{ number }}_{{ faiss_proc_type }}"
+      run_exports:
+        # faiss follows SemVer, so restrict packages built with libfaiss to use
+        # at least the same maj_min at runtime, but below next major version.
+        - {{ pin_compatible('libfaiss', min_pin='x.x', max_pin='x') }}
```

It's better to have `min_pin=x.x.x.x.x.x` which is the default to avoid unexpected surprises. 
```suggestion
        - {{ pin_compatible('libfaiss', max_pin='x') }}
```

---

I like to be more explicit (because someone who doesn't know the defaults of `pin_compatible` is stuck wondering or searching what that means exactly), but I just added an extra comment-line